pkg/compactor/blocks_cleaner.go

```diff
@@ -35,11 +35,12 @@ const (
 )
 
 type BlocksCleanerConfig struct {
-	DeletionDelay           time.Duration
-	CleanupInterval         time.Duration
-	CleanupConcurrency      int
-	TenantCleanupDelay      time.Duration // Delay before removing tenant deletion mark and "debug".
-	DeleteBlocksConcurrency int
+	DeletionDelay             time.Duration
+	PartialBlockDeletionDelay time.Duration
```

Comment:
This field is not used, and can be removed.

Reply:
Thanks for catching that